gnosis/safe-react
# fix: virtualize collectibles list

## What it solves
Large collectible lists freezing the Safe

## How this PR fixes it
The collectibles list is now virtualized, rendering only what is visible in the view port. It no longer freezes the Safe and is much more performant.

## How to test it
Open the Collectibles section of a Safe with a large number of collectibles, e.g. `eth:0xc6400A5584db71e41B0E5dFbdC769b54B91256CD` (prod.) and observe no lag in navigating the list.

## Screenshots
![image](https://user-images.githubusercontent.com/20442784/159003227-6b3545c3-b417-46fe-a761-ff1755626cc2.png)
